### PR TITLE
PE-4430: Translations update

### PR DESCRIPTION
**Pull request by matibat**
Exported the most recent translations from Localizely

--- Releases ---
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/3v6seb665jhu8

### CODE REVIEW

File: lib/l10n/app_en.arb [hunk 1]
```diff
@@ -310,7 +310,7 @@
   "@create": {
     "description": "Create an drive"
   },
-  "createAndConfirmPassword": "Please create and confirm your password. Your password cannot be changed or retrieved, so please keep this safe.",
+  "createAndConfirmPassword": "Please create and confirm your password.",
```
Comment (matibat) [inline lib/l10n/app_en.arb:313]: Was this intended? I think yes. Right, @thiagocarvalhodev?

File: lib/l10n/app_zh-HK.arb [hunk 2]
```diff
@@ -1,5 +1,11 @@
 {
-  "@@locale": "zh_HK",
+  "@@locale": "zh-HK",
```
Comment (matibat) [inline lib/l10n/app_zh-HK.arb:2]: Revert
```suggestion
  "@@locale": "zh_HK",
```

**Review by matibat — COMMENTED** [review]
A couple of comments

File: lib/l10n/app_es.arb [hunk 3]
```diff
@@ -1304,6 +1412,8 @@
   "@publicDrives": {
     "description": "Public drives accordion title"
   },
+  "quoteUpdatesIn": "Actualizaciones de presupuesto en",
```
Comment (matibat) [inline lib/l10n/app_es.arb:1415]: Original
> Quote updates in 

Translation: 
> Actualizaciones de presupuesto en

Mistakes:
1. Missing the space at the end
2. I'd translate it to: "El presupuesto se actualiza en "

Comment (matibat) [reply lib/l10n/app_es.arb:1415]: Why?
> Quote updates

Is equivalent to "It updates" -> "Se actualiza". It is a verb in the present tense, and it's singular. The translation they made is: "The update of the quotes in"

File: lib/l10n/app_en.arb (hunk 1 above)
Comment (matibat) [reply lib/l10n/app_en.arb:313]: This is correct. Confirmed by the team 👍

File: lib/l10n/app_es.arb (hunk 3 above)
Comment (matibat) [reply lib/l10n/app_es.arb:1415]: UPDATE: I just checked and this issue is present in the rest of the languages as well. We're gonna make the localization to consider the seconds as a variable to address this issue.

File: lib/l10n/app_es.arb [hunk 4]
```diff
@@ -58,6 +68,10 @@
       }
     }
   },
+  "arBalance": "Saldo",
```
Comment (matibat) [inline lib/l10n/app_es.arb:71]: In Spanish, "Saldo" and "Balance" both mean "funds"  or "balance". We're now using both words mixed.

File: lib/l10n/app_es.arb (hunk 3 above)
Comment (matibat) [reply lib/l10n/app_es.arb:1415]: Being fixed at #1297

**Review by thiagocarvalhodev — APPROVED** [review]

**Comment by matibat** [comment]
Closed because we've mistakenly merged it in #1310. Now integrated into #1303.

**Pull request closed without merging**